charts/openstack-cluster/values.yaml

```diff
@@ -214,8 +216,28 @@ osDistro: ubuntu
 #
 # API server authentication/authorization webhook. Set this to
 # integrate into KubeadmControlPlane and KubeadmConfigTemplate
-# possible values: k8s-keystone-auth
-# authWebhook: k8s-keystone-auth
+# possible values: none, k8s-keystone-auth, azimuth-authorization-webhook
+authWebhook: none
+
+azimuthAuthorizationWebhook:
+  server: https://127.0.0.1/authorize
+  webhookVersion: v1
+  timeout: 10s
+  failurePolicy: Deny
+  additionalAuthorizers:
+    - type: Node
+      name: node
+    - type: RBAC
+      name: rbac
+  tls:
+    enabled: false
```

Comment:
Maybe, lets make this true, and just make it clear you really need to specify the cert for anything to work? Not sure.